config/locales/hi.yml

```diff
@@ -0,0 +1,41 @@
+
+hi:
+  errors:
+    messages:
+      taken_in_past: पहले लिया गया 
+      equal_to_current_password: इससे अलग पासवर्ड होना चाहिए 
+      password_complexity:
+        digit:
+          one: एक अंक होना चाहिए 
+          other: कम से कम %{count} अंक होने चाहिए  
+        lower:
+          one: एक लोअर-केस अक्षर होना चाहिए 
+          other: कम से कम %{count} अक्षर होने चाहिए  
+        symbol:
+          one: एक प्रतीक चिन्ह होना चाहिए 
+          other: कम से कम %{count} प्रतीक चिन्ह होने चाहिए  
+        upper:
+          one: एक अपर-केस अक्षर होना चाहिए 
+          other: कम से कम %{count} अपर-केस अक्षर होने चाहिए  
+  devise:
+    invalid_captcha: कॅप्टचा सही नही था 
+    invalid_security_question: सुरक्षा उत्तर गलत था 
+    paranoid_verify:
+      code_required: सपोर्ट टीम द्वारा दिया गया कोड डाले 
+    paranoid_verification_code:
+      show:
+        submit_verification_code: वेरिफिकेशन कोड डाले 
+        verification_code: वेरिफिकेशन कोड 
+        submit: सबमिट 
+    password_expired:
+      updated: नया पासवर्ड संरक्षित किया 
```

Comment:
<img width="1282" alt="Screen Shot 2020-10-08 at 10 54 06 AM" src="https://user-images.githubusercontent.com/1635820/95475784-982e3500-0954-11eb-8ff4-b63bcc3c9236.png">

Not sure if this is right? The English version is `Your new password is saved.`.